src/client/src/pages/DataView360/View/components/Adoptions.js

```diff
@@ -37,7 +37,13 @@ class Adoptions extends Component {
 
     getLatestPets(animals) {
         const latestPets = _.sortBy(animals, animal => {
-            return animal.Events.Time
+            if (typeof animal.Events === 'undefined' || typeof animal.Events.Time  === 'undefined' ){
+                return 1
+            }
+            else{
+                return animal.Events.Time
+            }
```

Comment:
Just to tighten it up a bit, I'd suggest replacing lines 40-45 with:
`return (animal.Events && animal.Events.Time) ? animal.Events.Time : 1 `

Reply:
Way more elegant!